tensorboard/components/vz_line_chart/vz-line-chart.ts

```diff
@@ -482,8 +501,27 @@ class LineChart {
     nanDisplay.symbol(Plottable.SymbolFactories.triangle);
     this.nanDisplay = nanDisplay;
 
-    return new Plottable.Components.Group(
-        [nanDisplay, scatterPlot, smoothLinePlot, group]);
+    const groups = [nanDisplay, scatterPlot, smoothLinePlot, group];
+    if (marginAccessors) {
```

Comment:
If `!marginAccessors`, can we set `this.marginAccessors` to `[]` instead of `undefined`? This is the semantically correct value, and reduces the amount of extra logic required (see comments below).

(Probably the best way to implement this is to write `marginAccessors = marginAccessors || []` at the top of this function or right before this `if`-statement, which could then be dissolved.)